### PR TITLE
added reserved variable sub-section. Plus, removed deprecated text.

### PR DESCRIPTION
**Pull request by romansprykee**
## PR Description

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by AymenSegni — COMMENTED** [review]
Please let's have an additional warning about the change in the SPRYKER_ vars, and the access will be denied to these variables,  many customers they already use the variables and they will suddenly has no access to the vars

File: docs/cloud/dev/spryker-cloud-commerce-os/add-variables-in-the-parameter-store.md [hunk 1]
```diff
@@ -58,6 +58,12 @@ Path examples:
 
 * /deans_jeans/prod/config/app/public/mail_host
 
+### Reserved variables
+Reserved variables are variable names that have special meaning in Spryker Cloud Commerce OS. Since their names are reserved, you cannot define your own variables using these names.
+The reserved variable names are the following:
+* `DUMMY_INIT`
+* `SPRYKER_*`
+
```
Comment (tvaleriy) [inline docs/cloud/dev/spryker-cloud-commerce-os/add-variables-in-the-parameter-store.md:66]: Could you wrap this into the warning block please?

**Review by tvaleriy — CHANGES_REQUESTED** [review]

File: docs/cloud/dev/spryker-cloud-commerce-os/add-variables-in-the-parameter-store.md (hunk 1 above)
Comment (romansprykee) [reply docs/cloud/dev/spryker-cloud-commerce-os/add-variables-in-the-parameter-store.md]: Done. I put content in a warning block.

**Comment by romansprykee** [comment]
@AymenSegni I added a comment in the warning block regarding existing variables that are using restricted names.

**Review by tvaleriy — DISMISSED** [review]

**Review by AymenSegni — APPROVED** [review]

**Review by hatemzidi-pro — APPROVED** [review]
LGTM

**Review by lenadoc — APPROVED** [review]

**Pull request merged**